semantics.json

```diff
@@ -142,7 +142,7 @@
       {
         "label": "Language mode label",
         "name": "languageModeLabel",
-        "default": "Change language mode",
+        "default": "Swap language",
```

Comment:
Swap language or swap languages?

```suggestion
        "default": "Swap languages",
```

Reply:
Good question 🤔 What do you think?

Reply:
Used languages

Reply:
I think maybe plural as we're swapping to things